app/src/main/java/com/example/mobilemovieapp/dao/iMovieDAO.kt

```diff
@@ -0,0 +1,13 @@
+package com.example.mobilemovieapp.dao
+
+import android.telecom.Call
+
+//import retrofit2.Call
+//import app.plantdiary.individualassignment304832.dto.Country
+//import retrofit2.http.GET
+
+interface iMovieDAO
+{
+    //@GET("/b/6227c3b9a703bb674925773a")
+    //fun getAllCountries() : Call<ArrayList<Country>>
```

Comment:
Avoid pushing commented code to Version Control.  Version control has tools that makes this unnecessary.
- You can look at history to see what used to be there.
- You can use branches for experimental features.